app/javascript/Components/Result/dropdown_menu.jsx

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import safe_marked from "../../common/safe_marked";
+
+export class DropDownMenu extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {onHover: false};
+  }
+
+  handleMouseEnter = () => {
+    this.setState({onHover: true});
+  };
+
+  handleMouseLeave = () => {
+    this.setState({onHover: false});
+  };
+
+  render() {
+    return (
+      <li
+        className="dropdown_menu"
+        onMouseEnter={this.handleMouseEnter}
+        onMouseLeave={this.handleMouseLeave}
+        onMouseDown={e => e.preventDefault()}
+      >
+        <div className="dropdown-header">{this.props.header}</div>
+
+        {this.state.onHover && (
+          <div className="list">
```

Comment:
On the flip side to my above comment, you don't need to add this class name, as it isn't used for styling or functional purposes.